rpm/fits/package.spec

```diff
@@ -55,15 +56,33 @@ cp log4j.properties version.properties %{buildroot}/usr/share/fits/
 cp %{_sourcedir}/fits-nailgun.service %{buildroot}/usr/lib/systemd/system/fits-nailgun.service
 cp -rf tools/ffident  %{buildroot}/usr/share/fits/tools/
 cp -rf tools/droid  %{buildroot}/usr/share/fits/tools/
+cp -rf %{_etcdir}/sysconfig/fits  %{buildroot}/etc/sysconfig/fits
 
 
 %post
 touch /var/log/archivematica/fits.log
 chown archivematica.archivematica /var/log/archivematica/fits.log
-# TODO: reload only when the script changes. Check https://fedoraproject.org/wiki/Packaging:Scriptlets?rd=Packaging:ScriptletSnippets#Systemd
 systemctl daemon-reload
+systemctl restart fits-nailgun.service
```

Comment:
RPM packages don't usually enable their services ( [source](https://fedoraproject.org/wiki/EPEL:SysVInitScripts?rd=Packaging:SysVInitScript#Why_don.27t_we....) ). Could you remove this change?

Reply:
Ready!